WinRT.Runtime/IInspectable.cs

```diff
@@ -16,7 +16,7 @@ public enum TrustLevel
     // IInspectable
     [ObjectReferenceWrapper(nameof(_obj))]
     [Guid("AF86E2E0-B12D-4c6a-9C5A-D7AA65101E90")]
-    public class IInspectable
+    public partial class IInspectable
```

Comment:
partial only to combine with IInspectable.net5.cs?  #if NET5_0 might be simpler

Reply:
Given that IInspectable implements an extra interface in .NET 5.0, we'd have to add quite a few `#if NET5_0` lines throughout the file. It's easier IMO to have two separate files.